misk/src/main/java/digester/service.proto

```diff
@@ -0,0 +1,84 @@
+syntax = "proto3";
+
+option java_multiple_files = true;
+option java_outer_classname = "DigesterServiceProtos";
```

Comment:
Keep the `option java_package = "com.squareup.digester.protos.service";` from the original proto file.